<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertiesTab</class>
 <widget class="QWidget" name="PropertiesTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PropertiesTab</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="widget">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="projectNameLabel">
       <property name="text">
        <string>Project name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="projectNameEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="abbreviationLabel">
       <property name="text">
        <string>Abbreviation</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="abbreviationEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="versionLabel">
       <property name="text">
        <string>Version</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="versionEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="languageLabel">
       <property name="text">
        <string>Language</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="languageComboBox"/>
     </item>
     <item row="4" column="1">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="problemLabel">
       <property name="text">
        <string>problem</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPlainTextEdit" name="problemEdit"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="sytemLabel">
       <property name="text">
        <string>system</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPlainTextEdit" name="systemEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description of</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
